src/ImageSharp/Metadata/Profiles/IPTC/IptcProfile.cs

```diff
@@ -20,6 +21,16 @@ public sealed class IptcProfile : IDeepCloneable<IptcProfile>
 
         private const uint MaxStandardDataTagSize = 0x7FFF;
 
+        /// <summary>
+        /// 1:90 Coded Character Set.
+        /// </summary>
+        private const byte IptcEnvelopeCodedCharacterSet = 0x5A;
+
+        /// <summary>
+        /// This value marks that UTF-8 encoding is used in application records.
+        /// </summary>
+        private static readonly byte[] CodedCharacterSetUtf8Value = { 0x1B, 0x25, 0x47 };
```

Comment:
```suggestion
        // Uses C#'s optimization to refer to the data segment in the assembly direclty, no allocation occurs.
        private static ReadOnlySpan<byte> CodedCharacterSetUtf8Value => new byte[]{ 0x1B, 0x25, 0x47 };
```
And update `WriteRecord` (L294) to take a ROS<byte>.